css/properties/justify-self.json

```diff
@@ -3,22 +3,23 @@
     "properties": {
       "justify-self": {
         "__compat": {
+          "description": "Supported in Grid Layout",
```

Comment:
I freely admit that I don't know much about Grid, so can you help me understand the meaning of this change?

Is this synonymous with basic support (i.e., if a browser recognizes `justify-self` and considers it in the cascade, then it's necessarily doing something to do with Grid layout)? Or are you saying that the data below is a specific feature of `justify-self` (i.e, that there are uses for `justify-self` that are not related to Grid but this data only covers the Grid uses)? I get the impression (from places like the [the box alignment spec](https://www.w3.org/TR/css-align-3/#justify-self-property) that it could be the latter, but I want to make sure I understand what you intended first.

Reply:
There is no justify-self in Flexbox, and it hasn't been implemented anywhere else yet. However as we are clarifying the others it seemed to make sense to clarify that what we are describing is grid layout support. Even though the spec says you can use this in block layout, reality is you can't.

We could add to each of these the section for block layout and abspos, with it all marked as not available, or we could just show grid since grid has it.

Reply:
Thank you, Rachel! That helps a lot. One more follow up question: do you think it's likely that `justify-self` will be implemented for non-Grid layouts?

If so, it might make sense to prepare for the future by structuring the data like this:

* Basic support (`__compat`)
  * \<the data you have in this PR\>
  * Feature: Supported in Grid Layout
    * \<the data you have in this PR\>
  * [later] Feature: some other feature of `justify-self`
    * \<new, different data\>

If not, then I've got a small suggestion for the description.

Either way, thank you again for your insight on this!

Reply:
it's in the spec so one would hope so :)

Reply:
haha OK, then if you wouldn't mind restructuring it as shown, we can get this merged. Thank you again! 😃

Reply:
I'm not sure this makes sense to have basic support, then duplicate it for grid layout. We only have support in grid layout, it isn't something that makes sense alone. It's only ever going to be implemented within other layout methods.

If you also want a basic support section we'd need to do that for all of the others too, but in my opinion that doesn't make sense. What does "basic support" mean in this context?